src/main/java/org/reaktivity/nukleus/tls/internal/stream/ClientStreamFactory.java

```diff
@@ -1401,30 +1440,18 @@ private void doReset(
     private void doCloseInbound(
         final SSLEngine tlsEngine) throws SSLException
     {
-        try
-        {
-            tlsEngine.closeInbound();
-        }
-        catch (SSLException ex)
-        {
-            // Inbound closed before receiving peer's close_notify: possible truncation attack?
-            // this remote end-point behavior is allowed by TLS RFC
-            final String message = ex.getMessage();
-            if (!message.contains("possible truncation attack"))
-            {
-                throw ex;
-            }
-        }
+        tlsEngine.closeInbound();
     }
 
     private void doCloseOutbound(
         SSLEngine tlsEngine,
         MessageConsumer networkTarget,
-        long networkId) throws SSLException
+        long networkId,
+        Runnable applicationCleaner) throws SSLException
```

Comment:
Rename `applicationCleaner` to `networkReplyDoneHandler`.

Reply:
done